addons/l10n_in_edi/data/account_edi_data.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="edi_in_einvoice_json_1_03" model="account.edi.format">
-        <field name="name">E-Invoice (IN)</field>
+        <field name="name">GST E-Invoice (India)</field>
```

Comment:
May be a different commit message for changes in `l10n_in_edi`? Doesn't related to `account_edi`